Makefile

```
@@ -1,19 +1,33 @@
-# NSED Orchestrator Development Makefile
-# Provides convenient access to all development commands
+# NSED Development Makefile
+# Workspace: nsed-agent-sdk (MIT) → nsed-agent (BSL) → nsed-orchestrator (BSL) + nsed-benchmarks (Apache 2.0)
 
-.PHONY: help setup test test-coverage test-unit test-integration clean db-status db-reset db-connect dev run logs ci install-hooks auth-token auth-check auth-setup audit typos
+.PHONY: all help setup test test-coverage test-unit test-integration clean db-status db-reset db-connect dev run logs ci install-hooks auth-token auth-token-new auth-check auth-setup audit typos
+.PHONY: check-sdk test-sdk check-agent test-agent run-agent dashboard check-orch test-orch
 
 # Default target
+all: help
 help: ## Show this help message
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (22).

(maxbodylength)